1.13/base/Dockerfile

```diff
@@ -1,6 +1,9 @@
 FROM        debian:stable
 MAINTAINER  The Prometheus Authors <prometheus-developers@googlegroups.com>
 
+ENV NODE_VERSION 10.15.2~dfsg-2
+ENV YARN_VERSION 1.19.0-1
```

Comment:
Why do we need to specify a version? The upstream [install instructions](https://yarnpkg.com/en/docs/install#debian-stable) make it seem like it's fine to just pull in the latest stable automatically.

Reply:
Yeah, we can install it without explicit version constraint, but I thought it'd be wise to pin versions for anything that produces our final output binaries.

Reply:
The only down side is that it creates an additional maintenance burden to keep the builder image up to date. We already spend some effort having to keep our Golang pinning up to date.

Reply:
Yep, that's the tradeoff. I don't care strongly either way... if you prefer removing the version constraints that's fine with me as well.

Reply:
I'm in favor of "let's try unconstrained, and add a lock if we have issues later".

@simonpasquier might also have an opinion here.

Reply:
I'm fine with this option.

Reply:
Alrighty, removed the node & yarn version pinning.